src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -3712,7 +3712,7 @@ void Interpolate::InterpolateRefExecutor::pillowRef(const uint8_t *in_ptr_, uint
         uint8_t *xpass_out_ptr_nc = nullptr;
         const uint8_t *ypass_in_ptr_nc = nullptr;
         if (xPass && yPass) {
-            size_t threadsNum = parallel_get_num_threads();
+            size_t threadsNum = parallel_get_max_threads();
```

Comment:
1. As far as I understand the problem with OMP `parallel_get_num_threads` is that it returns 1 being called outside parallel region. But here it seems to be called inside, so why we need  to change it?
2. What value returns `parallel_get_max_threads` in case we explicitly limit number of inference threads via plugin config?  In case of TB threading TBB Arena return correct (limited value), but not sure if the same is true for OMP.

Reply:
1. Actually they return the same value in tests, but I agree, that using of `parallel_get_num_threads` here is the right way. Will be reverted.
2. Yes, OMP does not keep threads limit. `parallel_get_max_threads` can return `inference_num_threads`, cores number or 2*cores in scope of one inference. I suppose we need to disable dynamic behavior for `inference_num_threads` flag.
Please note that this PR fixes the functional tests only. Further investigation will be done during performance analysis.